homework-12/homework-12.2/homework-12.2/SecondViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SecondViewController.swift
+//  homework-12.2
+//
+//  Created by Kirill Skrytskii on 18.04.22.
+//
+
+import UIKit
+
+class SecondViewController: UIViewController {
+
+    @IBOutlet weak var label: UILabel!
+    @IBOutlet weak var thirdLabel: UILabel!
+    @IBOutlet weak var secondLabel: UILabel!
```

Comment:
It's better to give these fields more distinctive names. For example, `nameLabel`